src/Symfony/Component/HttpFoundation/HeaderBag.php

```diff
@@ -190,7 +190,7 @@ public function remove(string $key)
      *
      * @throws \RuntimeException When the HTTP header is not parseable
      */
-    public function getDate(string $key, \DateTime $default = null)
+    public function getDate(string $key, \DateTimeInterface $default = null)
```

Comment:
This will break code that overrides this method.

Reply:
reverted